04_numeronym/NimishMudhale/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func numeronym(vals ...string) []string {
+	numeronymArray := make([]string, len(vals))
+	for i, val := range vals {
+		target := strings.Trim(val, "\n\t ")
+		len := len(target)
```

Comment:
Change this variable name.
Don't overwrite native functions. This prevents you from using the `len` function anywhere else in this variables scope. Not much of an issue in a local `for` loop, very bad for large scope (luckily the compiler catches it if you try and make it global).

Reply:
done